config.py

```diff
@@ -410,6 +410,72 @@
             '.*[0-9]+.*',
         ],
     ),
+
+    nb = dict(
+        lang_name = 'Norsk (bokmål)',
+        lang_dir = 'ltr',
+        database = 'nowiki_p',
+        wikipedia_domain = 'no.wikipedia.org',
+
+        beginners_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder',
+        beginners_link_title = 'Wikipedia:Bruk av kilder',
+        lead_section_policy_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder#Hvorfor_siterer_vi_kilder',
+        lead_section_policy_link_title = 'Wikipedia:Bruk_av_kilder',
+
+        citation_needed_category = 'Artikler som trenger referanser',
```

Comment:
Category names need to use underscores instead of spaces: `Artikler_som_trenger_referanser`, please.